src/nls/messages_zh-Hans.json

```diff
@@ -183,7 +183,7 @@
     "dashboard.pipelineRun.notFound": "未找到 PipelineRun",
     "dashboard.pipelineRun.pipelineTaskName.retry": "{pipelineTaskName}（重试 {retryNumber, number}）",
     "dashboard.pipelineRun.stepCompleted": "步骤已完成",
-    "dashboard.pipelineRun.stepCompleted.withError": "步骤已完成，退出代码为 {exitCode}",
+    "dashboard.pipelineRun.stepCompleted.exitCode": "",
```

Comment:
Should we keep this translation since the string itself hasnt changed?

Reply:
absolutely, forgot to restore it after the second rename. Thanks